packages/map-template/src/components/Wayfinding/Wayfinding.jsx

```diff
@@ -417,7 +425,7 @@ function Wayfinding({ onStartDirections, onBack, directionsToLocation, direction
                         {searchResults.map(location =>
                             <ListItemLocation
                                 key={location.id}
-                                icon={location.properties.type === 'google_places' ? googlePlacesIcon : undefined}
+                                icon={(location.properties.type === 'google_places' || location.properties.type === 'mapbox_places') ? googlePlacesIcon : undefined}
```

Comment:
Should we rename the `googlePlacesIcon` since it is now no longer exclusively used in a Google context?

Reply:
Good catch, I renamed it to `locationMarkerIcon`. Do you have any other suggestion?

Reply:
Maybe something like `externalLocationIcon` to indicate that it is used for only "external" locations? 🤔

Reply:
Good idea